infra/azure/modules/batch/main.tf

```diff
@@ -56,16 +56,6 @@ resource "azurerm_shared_image" "batch_worker_22_04" {
   }
 }
 
-resource "kubernetes_secret" "batch_worker_ssh_public_key" {
-  metadata {
-    name = "batch-worker-ssh-public-key"
-  }
-
-  data = {
```

Comment:
Did we have to generate this key file ourselves? Where were those instructions / commands if yes?

Reply:
Good catch, I removed the instruction to generate the key.